existing-communities/README.md

```diff
@@ -1 +1,47 @@
+
+### Communication
+Communication is perhaps the most important factor when it comes to collaborative software development. Much communication is traditionally done via e-mail, instant messaging, phone, web meetings and groupware {{"thissen2007communication" | cite}}. However as of late new, more enhanced tools like Slack[^slackwebsite] have emerged. With such new tools, it is possible to integrate updates from other development tools such as GitHub and make the visible to the whole team. A conclusion drawn by Kalliamvakou is that the use of GitHub and its tools reduce the need for communication and coordination needs. However challenges remain in collaborating with non-technical persons because commercial projects still resort to external tools outside GitHub {{"Kalliamvakou:2015:OSC" | cite}}.
+
+### Version control
+Many software projects makes use of version control. According to the annual Eclipse Community service this percentage totals to almost eighty percent (Git 43.9%, Subversion 30.7%, Other 5.8%) in 2014 {{"eclipse2014survey" | cite}}. From the statistics in the figure below it can be concluded that the Git has become the most popular version control method. The main reasons for the growing popularity of Git are its feature set and the usage in large projects such as the aformentioned Linux project {{"stephen2015git" | cite}}. Two of the most important features of Git are branching and pull-requests stimulate working together trough independent work {{"Kalliamvakou:2015:OSC" | cite}}.
+
+{% chart format="yaml" %}
+padding: 50
+data:
+    type: line
+    x: 'year'
+    columns:
+        - [year, 2011, 2012, 2013, 2014]
+        - [Git, 12.8, 32, 36.3, 43.9]
+        - [Subversion, 51.3, 46, 37.8, 30.7]
+        - [Mercurial & CVS, 17.9, 11.5, 8.1, 5.8]
+    axes:
+        data2: y2
+axis:
+    y2:
+        show: true
+title:
+    text: "Version control popularity by type"
+{% endchart %}
+
+### Issue tracking
+The quote "Software does not allow itself to be finished. Software is never finished" by Paul LeBlanc {{"leblanc1993writing" | cite}} shows the need for good issue tracking. There is an enormous list of issue-tracking solutions available {{"multiple2016issuetracking" | cite}}, each with their own strengths and weaknesses. Interviews conducted by Kalliamvakou show that for example the combination of the GitHub issue-tracker enhances self-organization when assiging tasks and resolving conflicts {{"Kalliamvakou:2015:OSC" | cite}}. However the GitHub solution is not optimal because the issue tracker lacks some critcial features such as severity, component and a bug report template {{"Zhou:2015:CAB:2745802.2745808" |  cite}}.
```

Comment:
`as severity, component and a bug report template`

This should all be pluralized I think, so `as severity, components and a bug report templates`